zerver/lib/cache.py

```diff
@@ -402,6 +403,15 @@ def flush_user_profile(sender, **kwargs):
     if kwargs.get('update_fields') is None or "alert_words" in kwargs['update_fields']:
         cache_delete(realm_alert_words_cache_key(user_profile.realm))
 
+    # Invalidate realm-wide slash commands cache if any user in the realm has changed
+    # the short_name of bot or adds or deactivates a bot of type UserProfile.SLASH_COMMANDS
+    if user_profile.is_bot and user_profile.bot_type == UserProfile.SLASH_COMMANDS:
+        if kwargs.get('update_fields') is None or "is_active" in kwargs['update_fields']:
```

Comment:
OK.  In that case, can you please update the comment above so that it's consistent with the code?